### PR TITLE
Add links for ToS and Privacy Policy

### PR DESCRIPTION
**Pull request by dchw**
For once this is live: https://github.com/earthly/website/pull/4

### CODE REVIEW

**Review by alexcb — APPROVED** [review]
looks good as long as those URLs are supposed to be 404ing because they have yet to be uploaded.

**Comment by dchw** [comment]
The referenced PR isnt merged yet. I stole the links from there. This is in preparation for when that is live.

File: cmd/earth/main.go [hunk 1]
```diff
@@ -1318,7 +1318,7 @@ func (app *earthApp) actionRegister(c *cli.Context) error {
 	var interactiveAccept bool
 	if !app.termsConditionsPrivacy {
 		// TODO: add link when we have one
-		rawAccept := promptInput("Do you accept Earthly's T&Cs <link> and Privacy Policy <link>? [y/N]")
+		rawAccept := promptInput("Do you accept Earthly's Terms & Conditions (https://earthly.dev/tos) and Privacy Policy (https://earthly.dev/privacy-policy) [y/N]")
```
Comment (vladaionescu) [inline cmd/earth/main.go:1321]: To make our lawyers happy...

```suggestion
		rawAccept := promptInput("I acknowledge Earthly Technologies’ Privacy Policy (https://earthly.dev/privacy-policy) and agree to Earthly Technologies Terms of Service (https://earthly.dev/tos)")
```

**Review by vladaionescu — CHANGES_REQUESTED** [review]

**Review by vladaionescu — APPROVED** [review]

**Comment by vladaionescu** [comment]
Oh might want to also remove the TODO above `// TODO: add link when we have one`

**Pull request merged**